third_party/terraform/website/docs/r/google_project_organization_policy.html.markdown

```diff
@@ -127,6 +127,8 @@ The `restore_policy` block supports:
 In addition to the arguments listed above, the following computed attributes are
 exported:
 
+* `id` - an identifier for the resource with format `projects/{{project}}:constraints/{{constraint}}`
```

Comment:
I don't think we necessarily want to publish the id format in every case. This is one case where the id is not particularly useful, as it does not correlate with a self_link. You can tell because of the non-standard form that includes a colon `:`.

I don't believe that a user should use the `id` field for this resource